doc/faq.rst

```diff
@@ -0,0 +1,62 @@
+Frequently Asked Questions
+==========================
+
+.. contents::
+  :local:
+  :backlinks: none
+
+Installation / integration
+--------------------------
+
+Why all my tests are raising a ``ConnectionError`` when using SpeculosBackend?
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+This indicates that :term:`Speculos` could not reach its server, generally
+because the emulator could not be started.
+
+With verbose options, you should be able to find in the logs the Speculos
+command line, which should look like:
+
```

Comment:
+ check if a speculos process is already started

Reply:
Indeed!